pandas/core/indexes/datetimes.py

```diff
@@ -607,14 +607,10 @@ def _fast_union(self, other, sort=None):
         else:
             return left
 
-    def _wrap_setop_result(self, other, result):
-        name = get_op_result_name(self, other)
-        return self._shallow_copy(result, name=name, freq=None, tz=self.tz)
-
     def intersection(self, other, sort=False):
```

Comment:
where is this doc-string hit? if its only for DatetimeIndex, then move the doc-string there (as you did for TDI)